go/src/koding/klient/machine/machine.go

```diff
@@ -0,0 +1,17 @@
+package machine
+
+import "errors"
+
+var (
+	// ErrMachineNotFound indicates that provided machine cannot be found.
+	ErrMachineNotFound = errors.New("machine not found")
+)
+
+// IsMachineNotFound is a helper function that checks if provided error
+// describes missing machine.
+func IsMachineNotFound(err error) bool {
+	return err == ErrMachineNotFound
```

Comment:
@cihangir Indeed, It might be... when everything is ready with `machine` subcommand (mounts, ssh, etc) I will check if these helpers are useful and, if not, remove all of them.